### ISSUE TITLE
Google Analytics v4 support

### ISSUE THREAD

**Issue by jacul** [Mitrichius/hugo-theme-anubis#84]
When adding Google Analytics, I found that the G-XXX measurement ID wouldn't work. Switching to Universal Analytics Property seems to do the trick.

Looking at the doc here: https://gohugo.io/templates/internal/  
It says "When using Google Analytics v4 use _internal/google_analytics.html."

I believe anubis theme is using the async one? Would it be possible to also support the sync one?

Thanks!

**Comment by Mitrichius** [comment]
Thanks, I didn't know about new GA version. 
Added support for this - just add G-measurement ID to config and use last hugo version

**Comment by jacul** [comment]
Thanks for the quick fix! I verified that it's picking up the right template now.

Just a note, for whatever reason, the internal template doesn't seem to work with GA4. I took a workaround from https://gist.github.com/zjeaton/42246742cdaf2fb46400d04c2eba9a8a#file-analytics-gtag-html and it worked pretty well. I don't think it should be patched into the theme though, just leaving it here for anyone who might also run into it.
